configs/full.yaml

```diff
@@ -99,10 +92,27 @@ npz_fixed_field_keys:
 # key_mapping:
 #   free_energy: total_energy
 
+# npz example 
+# the keys used need to be stated at least once in key_mapping, npz_fixed_field_keys or include_keys
```

Comment:
@nw13slx I think this could be made more clear, e.g. what happens if a user only supplies a key in include_keys, but nowhere else, what does it get mapped to?